docs/uenv-compilation-spack.md

```diff
@@ -0,0 +1,163 @@
+# Using uenvs as upstream Spack instances
+
+User-environments (uenvs) are built with [Spack] using the [Stackinator] tool. Therefore, an `uenv` is tightly copuled with [Spack] and can be used as an upstream [Spack] instance (see [Chaining Spack Installations] for more details).
```

Comment:
```suggestion
User-environments (uenvs) are built with [Spack] using the [Stackinator] tool. Therefore, a `uenv` is tightly coupled with [Spack] and can be used as an upstream [Spack] instance (see [Chaining Spack Installations] for more details).
```

General question which doesn't need to be answered here: Is there a distinction between
- user-environment
- uenv
- `uenv`

?

Naively I'd say the first is a generic term, same as user environment. uenv: project name or a squashfs image? `uenv`: name of the command line tool?